ee/clickhouse/models/action.py

```diff
@@ -93,7 +94,7 @@ def format_entity_filter(entity: Entity, prepend: str = "action", filter_by_team
             action = Action.objects.get(pk=entity.id)
             entity_filter, params = format_action_filter(action, prepend=prepend, filter_by_team=filter_by_team)
         except Action.DoesNotExist:
-            raise ValueError("This action does not exist")
+            raise ValidationError("This action does not exist")
```

Comment:
One suggestion here is to add a `code` param to provide a machine-friendly code representation that can then help frontend do special handling (or better API processing for users who connect directly). Example:

```python
raise ValidationError("This action does not exist", code="does_not_exist")
```

would output:
```json
{
  "type": "validation_error",
  "code": "does_not_exist",
  "detail": "This action does not exist",
  "attr": null
}
```